feature-libs/product/configurators/common/src/components/config-message/config-message.component.ts

```diff
@@ -16,8 +16,9 @@ const updateMessageElementId = 'cx-config-update-message';
 })
 export class ConfigMessageComponent implements OnInit, OnDestroy {
   changesInProgress = false;
+  subscription: Subscription = new Subscription();
```

Comment:
Every property that you're not using in the template should be marked as `protected` (or `private` in special cases).

Reply:
Done